lib/yaml_elixir/exceptions.ex

```diff
@@ -3,5 +3,27 @@ defmodule YamlElixir.FileNotFoundError do
 end
 
 defmodule YamlElixir.ParsingError do
-  defexception message: "parsing error"
+  defexception [:line, :column, :type, message: "parsing error"]
+
+  @impl true
+  def message(%__MODULE__{message: message, line: nil, column: nil}) do
+    message
+  end
```

Comment:
You don't have to use `do...end` block in one-line functions. Please use `do: ...` syntax.